recipebox/recipebox/views.py

```diff
@@ -1,33 +1,81 @@
-from django.http import HttpResponse
-from django.shortcuts import render
-from recipebox.models import Author, Recipe
+from django.contrib.admin.views.decorators import staff_member_required
+from django.http import Http404
+from django.urls import reverse
+from django.shortcuts import redirect, render
+from .models import Author, Recipe, User
+from .forms import AuthorForm, RecipeForm
 
 
-def index_home(req):
-    recipes = Recipe.objects.all()
-    return render(req, 'index.html', {
-        'recipes': recipes,
-    })
+@staff_member_required
+def AuthorCreateView(req):
+
+    if req.method == "POST":
+        form = AuthorForm(req.POST)
 
+        if form.is_valid():
+            d = form.cleaned_data
+            user = User.objects.create_user(d['username'])
+            Author.objects.create(
+                user=user,
+                bio=d['bio'])
+            return redirect(reverse('home'))
 
-def authors(req):
+    else:
+        form = AuthorForm
```

Comment:
I learned something today! I thought this had to be instantiated in order to render, but it looks like it doesn't. Interesting!